src/Helper.ts

```diff
@@ -304,12 +318,12 @@ export class Helper {
       manifestGroup.addMetadata(manifestMetadata, true);
     }
 
-    if (this.manifest.getDescription().length) {
+    if (this.manifest.getSummary().length) {
       const metadataItem: LabelValuePair = new LabelValuePair(locale);
       metadataItem.label = new PropertyValue([
-        new LocalizedValue("description", locale),
+        new LocalizedValue("Summary", locale),
```

Comment:
I'm not sure how internationalization works here without reviewing the code, but I wonder if "description" is some kind of predetermined value that will get appropriately translated in some situations -- and if so, replacing it with summary may require translation work to be done somewhere else. Not sure if it's safer to leave the label alone for now or if we want to do a deep dive and figure out if/where to apply "summary" translations.